addons/base_setup/views/res_config_settings_views.xml

```diff
@@ -199,7 +199,7 @@
                         <div id="multi_company" groups="base.group_multi_company">
                             <field name="company_id" invisible="1"/>
                             <h2>Multi-Company</h2>
-                            <div class="row mt16 o_settings_container" name="multi">
+                            <div class="row mt16 o_settings_container" name="multicompany_setting_container">
```

Comment:
I think this should not be touched because it may have already been used

Reply:
Hmm perhaps not but the name was very meaningless. As it would land in the upcoming version the dev that migrates custom dev would notice it anyways? Not sure whichever Odoo itself prefers. I can change it if they want :)